db/migrate/20181113182615_add_vms_table.rb

```diff
@@ -0,0 +1,26 @@
+class AddVmsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :vms do |t|
+      t.references :tenant, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+
+      t.string :source_ref
+      t.string :uuid
+      t.string :vendor
+      t.string :name
+      t.string :hostname
+      t.string :description
+      t.string :power_state
+      t.bigint :cpus
+      t.bigint :memory
+
+      t.timestamps
+      t.datetime :archived_at
+      t.datetime :source_created_at
+      t.datetime :source_deleted_at
+
+      t.index [:source_id, :source_ref], :unique => true
+      t.index :archived_at
```

Comment:
should be ```archived_on```

Reply:
Yeah I saw that on the other models and I think we should change them to archived_at.  Every other timestamp is `_at`:
`created_at`, `updated_at`, `source_created_at`, and `source_deleted_at`